kong/license_service_test.go

```diff
@@ -0,0 +1,78 @@
+package kong
+
+import (
+	"os"
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestLicensesService(t *testing.T) {
+	RunWhenEnterprise(t, ">=2.3.0", RequiredFeatures{})
+	require := require.New(t)
+	assert := assert.New(t)
+	licenseString := os.Getenv("KONG_LICENSE_DATA")
```

Comment:
This test as far as I understand doesn't make sense when `KONG_LICENSE_DATA` is empty right? Maybe we 

- check that here and skip the test when that's the case
- add a similar helper func which would skip the test if the license is not precent, similar to `RunWhenEnterprise()`

WDYT?

Reply:
I think the Enterprise tests fail fast and run nothing when they have no license, but will check. That's probably CI-only though, so may as well add a code-level check.